frontend/src/assets/css/styles.scss

```diff
@@ -0,0 +1,39 @@
+@tailwind base;
+@tailwind components;
+@tailwind utilities;
+
+@layer base {
+    html {
+        font-family: Manrope, sans-serif;
+    }
+}
+
+@layer components {
+    h1 {
+        @apply text-4xl leading-9;
+    }
+
+    h2 {
+        @apply text-3xl leading-8;
+    }
+
+    h3 {
+        @apply text-2xl leading-7;
+    }
+
+    h4 {
+        @apply text-xl leading-6;
+    }
+
+    h5 {
+        @apply text-base leading-5;
+    }
+
+    body {
+        @apply text-sm leading-4;
+    }
+
+    caption {
+        @apply text-xs leading-3;
+    }
+}
```

Comment:
It might be better to simply expose classes for each typography style, such as .h1, .h2, body, body-sm etc.